release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -156,6 +156,22 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-9-olm"]
 === Operator lifecycle
 
+[id="ocp-4-9-fb-catalogs"]
+==== File-based catalogs
```

Comment:
Relnote for https://github.com/openshift/openshift-docs/pull/36352. Will update with an xref after that PR merges.